### PR TITLE
Modify getting-started document to reflect our new Azure project structures, testing, etc...

### PR DESCRIPTION
**Pull request by BradySkylight**
There are places where we still need to do work and they are marked by a ??TODO  or TODO??.

### CODE REVIEW

**Review by DanPaseltiner — APPROVED** [review]
This is great. A couple minor things I think we should address before merging, but no need for them to be blocking. Once we get a little more familiar with somethings in Azure like ADF we'll need to do another revision on this doc.

File: docs/README.md [hunk 1]
```diff
@@ -41,10 +41,16 @@ To deploy this pipeline to your own Azure environment, follow these steps.
...
+  1. Authenticate the Azure cloud CLI by running:
+       
+      for Unix based systems
 
          ./quick-start.sh
 
+      for Windows based systems
+      
+         quick-start.ps1
+
```
[the remark this answers is not in the document]
Comment (BradySkylight) [reply docs/README.md:53]: Yup, no problem.  Will do that now.

File: docs/getting_started.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
 # Getting Started
 
-This is a guide for getting started as a user and/or developer with the PRIME PHDI Google Cloud project. You'll find resources on how to setup a local development environment, how these tools are deployed, and more.
+This is a guide for getting started as a user and/or developer with the PRIME PHDI Azure project. You'll find resources on how to setup a local development environment, how these tools are deployed, and more.
```
[the remark this answers is not in the document]
Comment (BradySkylight) [reply docs/getting_started.md:3]: Love this!  Great idea!

File: docs/getting_started.md [hunk 3]
```diff
@@ -83,26 +87,64 @@ Until we have properly containerized our apps, we will need to rely on informal
...
- 
+2. Install the [Azure Core Function Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local), but be sure to install the 3.x version, and not 4.x. 4.x does not work with all Azure functionality as well as one might hope.
+3. Install the [.NET Core SDK 3.1](https://dotnet.microsoft.com/en-us/download/dotnet/3.1). Versions 5.0 and 6.0 do not seem to work well with the 3.x Azure Core Function Tools.
+4. Install [Python 3.9.x](https://www.python.org/downloads/).  As of this writing, this is the highest Python version supported by Azure Funcation Apps.
+5. Install [Azure CLI Tools](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).  These tools are essential to work with various aspects of the Azure cloud. 
+In the next section, "Developing Azure Locally", we'll go over which Extensions you need to install, including the Azurite Extension, in order to begin developing Azure functionality.
+6. Install [pip](https://pip.pypa.io/en/stable/installation/). This is the dependency manager for the Azure Functions
+7. Install [poetry](https://python-poetry.org/docs/). This is the dependency manager for the internal library.
```
[the remark this answers is not in the document]
Comment (BradySkylight) [reply docs/getting_started.md:102]: I pulled this from the previous azure project, so we may not need it.  I'll remove it for now, but we should review this doc again after we are done creating all the BBs.

File: docs/README.md [hunk 4]
```diff
@@ -64,7 +70,7 @@ The PHDI Building Blocks are implemented as Azure Function Apps. Azure Function
 
 #### Pipeline Orchestration
 
-Since the Building Blocks are designed to be composable users may want to chain serveral together into pipelines. We use the Google Workflow resource to define processes that require the use of multiple Building Blocks. These workflows are defined using YAML configuration files found in the `worklows` directory.
+Since the Building Blocks are designed to be composable users may want to chain serveral together into pipelines.  **TODO**
```
[the remark this answers is not in the document]
Comment (BradySkylight) [reply docs/README.md:73]: https://app.clickup.com/t/327fry3

File: docs/getting_started.md [hunk 5]
```diff
@@ -83,26 +87,64 @@ Until we have properly containerized our apps, we will need to rely on informal
...
+2. Creates `.vscode/tasks.json`, which make it easier to activate the relevant virtual environment, installs dependencies, and starts a debugging session
+3. Creates `.vscode/launch.json`, which makes it so that when you hit F5 / go to `Run->Start Debugging` it runs the tasks from (2) and then attaches to the debugger for your functions when running locally.
+
+#### Azure Function Directory Structure
+
+All Azure Functions live in the [severless-functions](https://github.com/CDCgov/phdi-azure/tree/main/serverless-functions) directory. The tree below shows a hypoethetical example for a Azure Function called `myfunction`. The PHDI team believes strongly in the importance of developing well tested code, so we include an additional file called `test_<FUNCTION-NAME>.py`. In this example `test_myfunction.py` cotains the unit tests for `myfunction`. The deployment process for `myfunction` does???  TODO??
 
 ```bash
-cloud-functions/
+serverless-functions/
```
[the remark this answers is not in the document]
Comment (BradySkylight) [reply docs/getting_started.md:147]: Interesting.  I was just going off of what we actually have within the project itself.

**Pull request merged**